Scale_space_reconstruction_3/include/CGAL/Scale_space_reconstruction_3/Scale_space_surface_reconstruction_3_impl.h

```diff
@@ -24,15 +24,30 @@
 #include "tbb/parallel_for.h"
 #endif // CGAL_LINKED_WITH_TBB
 
+#include <sstream>
+#include <fstream>
+
 #include <boost/function_output_iterator.hpp>
 #include <boost/lexical_cast.hpp>
 #include <string>
 
+#include <CGAL/Mesh_3/dihedral_angle_3.h>
+
+
 namespace CGAL {
 
-template < class Gt, class FS, class Sh, class wA, class Ct >
+template <typename T>
+struct operator_less
```

Comment:
Hi, 

Do we have such a tool ? 

Best 

-- 
Nicolas Douillet 
TITANE project team 
Byron Y310 

04 92 38 50 66 

https://team.inria.fr/titane